pdns/recpacketcache.cc

```diff
@@ -236,7 +261,7 @@ uint64_t RecursorPacketCache::doDump(int fd)
   for(auto i=sidx.cbegin(); i != sidx.cend(); ++i) {
     count++;
     try {
-      fprintf(fp, "%s %d %s  ; tag %d\n", i->d_name.toString().c_str(), (int32_t)(i->d_ttd - now), DNSRecordContent::NumberToType(i->d_type).c_str(), i->d_tag);
+      fprintf(fp, "%s %ld %s  ; tag %d\n", i->d_name.toString().c_str(), (i->d_ttd - now), DNSRecordContent::NumberToType(i->d_type).c_str(), i->d_tag);
```

Comment:
It's probably going to be more of an issue on 64bit platforms with a 32bit `time_t`, but indeed I should fix that!